### PR TITLE
CASMINST-5899 add troubleshooting info for when a ceph image has tag:<none>

### PR DESCRIPTION
**Pull request by leliasen-hpe**
# Description

CASMINST-5899. Add troubleshooting infro for when a ceph image has tag:<none>

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/utility_storage/Troubleshoot_ceph_image_with_none_tag.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Troubleshoot Ceph image with tag:'\<none\>'
+
+Use the following script to fix a ceph image with tag: \<none\>.
```
Comment (cdelatte-hpe) [inline operations/utility_storage/Troubleshoot_ceph_image_with_none_tag.md:3]:
```suggestion
Use the following procedure to fix a ceph image with tag: \<none\>.
```

**Review by cdelatte-hpe — DISMISSED** [review]
since this is a procedure it makes more sense than calling it a script

File: operations/utility_storage/Troubleshoot_ceph_image_with_none_tag.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            full_id=$(podman images $image_name --format json | jq '.[].Id' | grep $image_id | tr -d '"' | tail -1)
+            version=$(podman images $image_name --format json | jq --arg FULL_ID $full_id '.[] | select (.Id == $FULL_ID) | .Labels."org.opencontainers.image.version"' | tr -d '"' | tail -1)
+            podman pull ${image_name}:${version}
+        done
+        ```
+    
+    1. Change the mode of the script.
+
+        ```bash
+        chmod u+x /tmp/fix_ceph_image_tag.sh
```
Comment (bklei) [inline operations/utility_storage/Troubleshoot_ceph_image_with_none_tag.md:55]: Would it make sense to check this into the scripts directory?

**Review by bklei — APPROVED** [review]

File: operations/utility_storage/Troubleshoot_ceph_image_with_none_tag.md (hunk 2 above)
Comment (leliasen-hpe) [reply operations/utility_storage/Troubleshoot_ceph_image_with_none_tag.md]: I moved this to the scripts dir.

**Review by rgross727 — APPROVED** [review]

**Pull request merged**